### PR TITLE
Fixes #987

### LINKED ISSUE #987: support "OR" for adjust query flags in cartridge

### PR DESCRIPTION
**Pull request by greglandrum**

### CODE REVIEW

File: Code/PgSQL/rdkit/expected/rdkit-91.out [hunk 1]
```diff
@@ -771,3 +771,77 @@ select 'C1CC1C(C)OC'::mol @> mol_adjust_query_properties('C1CC1CO*'::mol,'{"adju
  f
 (1 row)
 
+select 'C1CC1CCCC'::mol @> mol_adjust_query_properties('C1CC1CC'::mol);
```
Comment (bp-kelley) [inline Code/PgSQL/rdkit/expected/rdkit-91.out]: Can mol here be a ctab?  If so, we should probably aromatize by default using the new adjustQuery prop?  this actually might be true of smiles as well actually.

Comment (greglandrum) [inline Code/PgSQL/rdkit/expected/rdkit-91.out]: You can provide mol using mol_from_ctab() or qmol_from_ctab(), but those aromatize by default (as does the SMILES parser) so it doesn't seem necessary:
```
chembl_21=# select molregno,m from rdk.mols where m@> qmol_from_ctab('
  Mrv1561 07261604572D          

  6  6  1  0  0  0            999 V2000
   10.4018   -8.7850    0.0000 L   0  0  0  0  0  0  0  0  0  0  0  0
    9.6873   -8.3725    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    9.6873   -7.5475    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   11.1162   -7.5475    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   11.1162   -8.3725    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    8.9729   -8.7850    0.0000 A   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0  0  0  0
  2  3  2  0  0  0  0
  4  5  2  0  0  0  0
  1  5  1  0  0  0  0
  3  4  1  0  0  0  0
  2  6  1  0  0  0  0
  1 F    2   7   8
M  ALS   1  2 F N   O   
M  END
') limit 4;
 molregno |                                                     m                                                      
----------+------------------------------------------------------------------------------------------------------------
   322267 | C=Cc1c(C)c2n3c1=CC1=[N+]4C(=Cc5c(C)c6c7n5[Zn-2]34[N+]3=C(C=2)[C@@H](C)[C@H](CCC(=O)OC)C3=C7CC6=O)C(CC)=C1C
   260055 | CNC(=O)CCc1c(SSc2[nH]c3ccccc3c2CCC(=O)NC)[nH]c2ccccc12
   260101 | CN(C)C(=O)CCc1c(SSc2[nH]c3ccccc3c2CCC(=O)N(C)C)[nH]c2ccccc12
   246719 | Cc1c(CC(N)=O)c2c(OCC(=O)[O-])cccc2n1Cc1cccc(Cl)c1.[Na+]
(4 rows)
```

Doing the query with `mol_from_ctab()` returns the same results (somewhat surprising at the moment, need to look into that, but that's not part of this PR).
